doc/NPCs.md

```diff
@@ -1432,6 +1434,7 @@ This section is a work in progress as functions are ported from `arithmetic` to
 |----------|------|--------|-------|-------------|
 | pain     |  ✅  |   ✅   | u, n  | Return or set pain<br/> Example:<br/>`{ "math": [ "n_pain()", "=", "u_pain() + 9000" ] }`|
 | skill    |  ✅  |   ✅   | u, n  | Return or set skill level<br/> Example:<br/>`"condition": { "math": [ "u_skill('driving')", ">=", "5"] }`|
+| game_option   |  ✅  |   ❌   | g     | Return the numerical value of a game option<br/> Example:<br/>`"condition": { "math": [ "game_option('NPC_SPAWNTIME')", ">=", "5"] }`|
```

Comment:
Should be sorted before pain. Also, I prefer `N/A<br/>global` for scope since it matches the prefix (and I wrote it that way for `weather`)